packages/react-core/single-packages.config.json

```diff
@@ -1,4 +1,5 @@
 {
     "packageName": "@patternfly/react-core",
-    "exclude": []
+    "moduleGlob": ["/dist/esm/helpers/**/*.js", "/dist/esm/*/*/**/index.js"],
```

Comment:
Hi @Hyperkid123 we have an [issue](https://github.com/patternfly/patternfly-react/issues/7932) open for our next major release (v6) to clean up some of the utilities/helpers that are exported since they are not all meant for external use.  We also would like to document the ones that are being used.  Can you please let me know which helpers you are using so we keep those exported for consumption.

Reply:
@tlabaj It's mainly the `Popper` component for custom menus.